plugins/org.eclipse.elk.core.meta/src/org/eclipse/elk/core/meta/jvmmodel/MetaDataJvmModelInferrer.xtend

```diff
@@ -93,31 +99,28 @@ class MetaDataJvmModelInferrer extends AbstractModelInferrer {
             // 1. Public constants for all declared properties
             for (property : bundle.members.allPropertyDefinitions) {
                 val constant = property.toPropertyConstant
-                if (property.defaultValue !== null)
+                if (property.defaultValue !== null) {
```

Comment:
We should either have a convention to always use braces (even in xtend files), or not change it in existing code.